### ISSUE TITLE
sorting results by distance from geo point?

### ISSUE THREAD

**Issue by jsebrien** [Stratio/cassandra-lucene-index#34]
Hello,

When performing a geodistance search, is it possible to sort results by increasing/decreasing distance from the geo point?

Regards,

Julien

**Comment by adelapena** [comment]
Not yet, but it's in our roadmap.

**Comment by jsebrien** [comment]
ok great, thanks

**Comment by ealonsodb** [comment]
Hi @jsebrien, We are currently developing this feature.

We have some work to do before including it in next release (query builder, proper tests, documentation) but you can take a look on the current work in branch [feature/sort_by_distance](https://github.com/Stratio/cassandra-lucene-index/tree/feature/sort_by_distance)

Here is a Query example: 

```

SELECT * FROM users WHERE lucene='{
    query  : { type         : "geo_distance",
               field        : "geo_point",
               latitude     : 40.442163,
               longitude    : -3.784519,
               max_distance : "10000km"},
    sort   : { fields: [ { type      : "geo_distance", 
    					   field     : "geo_point", 
    					   reverse   : false, 
    					   latitude  : 40.442163,
    					   longitude : -3.784519}]}
}' limit 10;

```
Cheers

**Comment by jsebrien** [comment]
This is a very good news. I will test this feature asap. thanks!

**Comment by ealonsodb** [comment]
This functionality was included in release [2.2.4.1](https://github.com/Stratio/cassandra-lucene-index/tree/2.2.4.1)

Thank you
